### DISCUSSION TITLE
Dropsets

### DISCUSSION THREAD

**Discussion by LiamMorrow** [LiamMorrow/LiftLog#831] [Ideas]
A lot of people like to do dropsets -> this is a form of pyramid set where the user starts with a high weight and goes to failure, then lowers the weight and goes to failure immediately again, as many times as they like.

It is currently possible to record these, but it is just building off of the "Per set weight" feature.

Per Set weight feature:

https://github.com/user-attachments/assets/de35e295-f800-4568-a2fb-514c3fa5e5bb

This issue is the starting point for designing a native dropset / pyramid set feature.  It could just look like a tutorial directing a user to the per set weight feature 🤔

**Comment by LiamMorrow** [comment]
Reddit comment:

https://www.reddit.com/r/opensource/comments/1e6syhw/liftlog_a_simple_workout_tracking_app/le5bdxf/

A feature that I realize was missing, because I've been doing it lately, is dropsets. It could be a toggle in the Add Exercise page, where you toggle it, now that exercise has dropset functionalities. To keep it simple, just select the number of sets, and display a list of weight and reps for each set. (Example: 3 sets -> 1st: 10 reps @ 40lb, 2nd: 10 reps @ 35lb, 3rd: 8 reps @ 30lbs). Just allow for custom weight input.

**Comment by JMulder99** [comment]
I don't see how the current implementation of the rep/weight setting per set does not allow one to easily fill in dropsets.

It doesn't make sense to pre-fill reps/weights per set for a dropsets, as the amounts depend on the performance in-situ.
